src/app/crear-pedido/create-pedido.tsx

```
@@ -1,35 +1,38 @@
-import {forma, mes, dia1, dia2, diaSemana, frecuencia} from "../utils/pedido_info"
-import {createPedido} from "../store/fetch-pedido"
+import React, { useState, useContext, useCallback } from 'react';
+import {Avatar, Box, Button, FormControl, Container, CssBaseline, InputLabel, Grid, MenuItem,
+  Select, TextField, Typography, Autocomplete, Paper, Divider, SelectChangeEvent} from '@mui/material';
+import BorderColor from '@mui/icons-material/BorderColor';
+import { Snack } from "../components/snackBar";
+import { forma, mes, dia1, dia2, diaSemana, frecuencia } from "../utils/pedido_info";
+import { createPedido } from "../store/fetch-pedido";
 import { usePathname, useRouter } from 'next/navigation';
-import {Date} from "../components/date"
+import { Date } from "../components/date";
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)